lib/samples/style_graphics_with_renderer/README.md

```diff
@@ -0,0 +1,42 @@
+# Style graphics with renderer
+
+A renderer allows you to change the style of all graphics in a graphics overlay by referencing a single symbol style. A renderer will only affect graphics that do not specify their own symbol style.
+
+![Image of add graphics with renderer](style_graphics_with_renderer.dart)
```

Comment:
"Image of add graphics with renderer" should be "Image of style graphics with renderer"

style_graphics_with_renderer.dart should be style_graphics_with_renderer.png

Reply:
Changed It.